### ISSUE TITLE
[playtime Session] select kids for new playtime session

### ISSUE THREAD

**Issue by Rushi-faldu** [amosproj/amos2024ws04-personalized-play#118]
### Purpose

The user should be able to select the kids that are present for this playtime activity session and also add a new kid.

### User story

As a user, I want the app to provide me an option to select kids present for a new playtime session so that I can be recommended an activity for present kids only.
As a user, I want the app to provide me an option to add a new kid so that I can add that data, and considering that i will be suggested some playtime session.

### Acceptance Criteria

- [ ] When the user starts a new playtime session (i.e., a second-time user), the system will display a question, "Select the kiddos keeping you busy today."
- [ ] the agent will display the radio list to select the kids present for this session.
- [ ] The user can select multiple kids at a time.
- [ ] By default, all kids will be selected
- [ ] The user has to select at least one kid to proceed.
- [ ] When the user clicks on the "Add new kid" button, the user will be directed to the "Give details about your kid" page of the contextual question.
- [ ] After that, the app will also ask the user for health considerations for the new kid, if any.
- [ ] Once the user is done with the health consideration question, the user will be redirected to the "select kids for a new playtime session" screen.
- [ ] Here, new kid data will be displayed in the radio list for names of kids.
- [ ] the system can add maximum 3 kids data.
- [ ] When the user clicks on the "next" button, the user will be redirected to the energy level question of contextual question flow.

Definition of Done (DoD)

- [ ] The feature has been fully implemented, thoroughly tested, and integrated into the app with no critical bugs or performance issues.

- [ ] Manual testing is complete, verifying that the feature meets all acceptance criteria.
 
- [ ] Big features are documented inside the code, with clear explanations of functionality, usage, and any relevant edge cases.
 
- [ ] The feature has undergone peer review, with the code approved by at least one other team member to ensure quality and maintainability.
 
- [ ] The feature branch has been successfully merged into the main branch, and the pull request is closed and the branch is deleted.
 
- [ ] The feature’s UI/UX has been reviewed to ensure consistency and alignment with design guidelines.
